types/underscore/index.d.ts

```diff
@@ -3533,39 +3545,44 @@ declare module _ {
         ): Extract<keyof V, string> | undefined;
 
         /**
-        * Return a copy of the object, filtered to only have values for the whitelisted keys
-        * (or array of valid keys).
-        * @param object Object to strip unwanted key/value pairs.
-        * @keys The key/value pairs to keep on `object`.
-        * @return Copy of `object` with only the `keys` properties.
-        **/
-        pick<T, K extends keyof T>(obj: T, ...keys: K[]): Pick<T, K>;
-        pick<T, K extends keyof T>(obj: T, keys: K[]): Pick<T, K>;
-        pick<T, K extends keyof T>(obj: T, predicate: ObjectIterator<T[K], boolean>): Pick<T, K>;
+         * Return a copy of `object` that is filtered to only have values for
+         * the whitelisted keys (or array of keys).
```

Comment:
In Underscore, we replaced "whitelisted" by "allowed" and "blacklisted" by "disallowed" (https://github.com/jashkenas/underscore/pull/2862). Would be neat to do that here as well while you're at it.

Reply:
Will do! Looks like that update just hasn't quite made it to http://underscorejs.org/ yet 😅

Reply:
Indeed it didn't! I'm currently working towards the 1.11 release, which will include it. And that's why I'm informing you here. 👍

Reply:
Thanks for doing so!